templates/msg.h.em

```diff
@@ -232,6 +237,9 @@ void _@(msg_underscored_name)_decode(const CanardRxTransfer* transfer, uint32_t*
 @[                  end if]@
 @{indent += 1}@{ind = '    '*indent}@
 @[              end if]@
+@(ind)if (msg->@(field.name).len > @(field.type.max_size)) {
+@(ind)    msg->@(field.name).len = @(field.type.max_size);
```

Comment:
i think we should change these void functions to bool. Unfortunately the convention is return true on invalid decode (that is completely crazy btw)

Reply:
> i think we should change these void functions to bool. Unfortunately the convention is return true on invalid decode (that is completely crazy btw)
The return true is to match the older compiler.